src/navigator/IFrameNavigator.ts

```diff
@@ -846,12 +854,12 @@ export default class IFrameNavigator implements Navigator {
     addEventListenerOptional(
       this.previousChapterAnchorElement,
       "click",
-      this.handlePreviousChapterClick.bind(this)
+      this.handlePreviousPageClick.bind(this)
```

Comment:
Just curious, why are you making this change here? And also the similar one below? I would think that if you had an event listener on the `previousChapterAnchorElement`, then you would want the callback to be `handlePreviousChapterClick` instead of `handlePreviousPageClick`.

Reply:
Yes you are right @pszujewski, i am trying to solve an issue where the arrow left and right in the material menu which are `previousChapterAnchorElement` and `nextChapterAnchorElement` work on mobile devices, as there is not much room to have the buttons in the middle, but maybe i'll add new buttons for mobile. i'll give it a try and see if i can come up with something that doesn't lead to confusion.

thanks for pointing that out :) 

<img width="401" alt="Screen Shot 2021-08-12 at 6 59 07 PM" src="https://user-images.githubusercontent.com/17150623/129280581-e825a6f7-be4b-4ac4-84b6-a18eab7a7094.png">

Reply:
reverted and changed the buttons to use api calls instead